functions/index.js

```diff
@@ -1,9 +1,65 @@
 const functions = require('firebase-functions')
+const axios = require('axios')
+const mailchimp = require('@mailchimp/mailchimp_marketing')
 
-// Create and Deploy Your First Cloud Functions
-// https://firebase.google.com/docs/functions/write-firebase-functions
+exports.processCalendlyEventInvitee = functions.https.onRequest(async (request, resp) => {
+  const { calendlyInviteeURI } = request.body
+  const token = 'eyJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJodHRwczovL2F1dGguY2FsZW5kbHkuY29tIiwiaWF0IjoxNjE4NjA3MDM3LCJqdGkiOiIxZjliMjRjOC03NGEwLTRlNzMtOGMxMy0xMGI0MjNlYmI2NjQiLCJ1c2VyX3V1aWQiOiJHR0hBUVVBQzNONUxUUlgyIiwib3JnX3V1aWQiOiJESENIVVJEQTZZWEJRUzZMIn0.UxvkeOZGkCiq_9RE6bU2PV-mD0keGKE7rwuvOZ2SZLY'
+  const listId = 'bfb4a12429'
 
-exports.addMemberToMailchimp = functions.https.onRequest((request, response) => {
-  functions.logger.info('Hello logs!', { structuredData: true })
-  response.send('OK')
+  resp.set('Access-Control-Allow-Origin', '*')
+  if (request.method === 'OPTIONS') {
+    resp.set('Access-Control-Allow-Methods', 'POST')
+    resp.set('Access-Control-Allow-Headers', 'Content-Type')
+    resp.set('Access-Control-Max-Age', '3600')
+    resp.status(204).send('')
+    return undefined
+  }
+
+  mailchimp.setConfig({
+    apiKey: '9ab820bcffa760d9f787083c910ddf6e-us1',
+    server: 'us1'
+  })
```

Comment:
- Move this outside of the function (to line 4).
- Also, extract the API Key and the server to constants in the global scope: `MAILCHIMP_API_KEY` AND `MAILCHIMP_SERVER`.